pages/common/git-browse-ci.md

```diff
@@ -0,0 +1,13 @@
+# git browse-ci
+
+> Opens the current `git` repository's CI website in the default web browser.
```

Comment:
```suggestion
> Opens Git repository's CI website in the default web browser.
```

Reply:
I think this is less grammatically correct.